src/display/canvas.js

```diff
@@ -1853,6 +1882,15 @@ class CanvasGraphics {
           break;
       }
     }
+
+    if (isScalingMatrix) {
+      Util.scaleMinMax(currentTransform, minMaxForBezier);
+      current.minX = Math.min(current.minX, minMaxForBezier[0]);
+      current.maxX = Math.max(current.maxX, minMaxForBezier[1]);
+      current.minY = Math.min(current.minY, minMaxForBezier[2]);
+      current.maxY = Math.max(current.maxY, [3]);
```

Comment:
This `[3]`-part looks like a simple typo, right?
If the patch passes all tests like this, then we'd probably want some better test-coverage here...

---

How about instead adding a new method for this code, rather than defining it inline like this?
Maybe something like the following (with perhaps a better name), placed e.g. somewhere [around here](https://github.com/mozilla/pdf.js/blob/379125c37ff58cfb86e3bbd2eca5760fb2fcb061/src/display/canvas.js#L612):
```javascript
updateScalingPathMinMax(transform, minMax) [
  Util.scaleMinMax(transform, minMax):
  this.minX = Math.min(this.minX, minMax[0]);
  this.maxX = Math.max(this.maxX, minMax[1]);
  this.minY = Math.min(this.minY, minMax[2]);
  this.maxY = Math.max(this.maxY, minMax[3]);
}
```